.gitignore

```diff
@@ -12,3 +12,4 @@ MANIFEST
 MANIFEST.bak
```

Comment:
*.bak files do not belong in the MANIFEST

Reply:
I use Git::GatherDir so it grabs only things in the repo - hence the reason it in .gitignore so I don't commit it